frontend/appflowy_flutter/test/unit_test/settings/shortcuts/settings_shortcut_service_test.dart

```diff
@@ -0,0 +1,120 @@
+import 'dart:convert';
+import 'dart:io' show File;
+import 'package:appflowy/workspace/application/settings/shortcuts/settings_shortcuts_service.dart';
+import 'package:appflowy_editor/appflowy_editor.dart'
+    show ShortcutEvent, builtInShortcutEvents;
+import 'package:flutter_test/flutter_test.dart';
+// ignore: depend_on_referenced_packages
+import 'package:file/memory.dart';
+
+void main() {
+  late SettingsShortcutService service;
+  late File mockFile;
+  String shortcutsJson = '';
+
+  setUp(() async {
+    final MemoryFileSystem fileSystem = MemoryFileSystem.test();
+    mockFile = await fileSystem.file("shortcuts.json").create(recursive: true);
+    service = SettingsShortcutService(passedFile: mockFile);
+    shortcutsJson =
+        """
+{
+"shortcuts": [
+{ 
+"key":"Home",
+"command":"alt+home"
+},
+{
+"key":"End",
+"command":"alt+end"
+},
+{
+"key":"Delete Text",
+"command":"alt+delete"
+}
+]
+}""";
```

Comment:
Could be prettier for readability

Reply:
Hey @Xazin, thank you, I wanted to make it more readable as well. But if I insert spaces and linebreaks, then they get recorded in the string and thus corrupt the json. So basically then the jsonDecode method doesn't work. Do you have any other idea to make it prettier for readability